beetsplug/playlist.py

```diff
@@ -86,6 +87,100 @@ class PlaylistPlugin(beets.plugins.BeetsPlugin):
     def __init__(self):
         super(PlaylistPlugin, self).__init__()
         self.config.add({
+            'auto': False,
             'playlist_dir': '.',
             'relative_to': 'library',
         })
+
+        self.playlist_dir = self.config['playlist_dir'].as_filename()
+        self.changes = {}
+
+        if self.config['relative_to'].get() == 'library':
+            self.relative_to = beets.util.bytestring_path(
+                beets.config['directory'].as_filename())
+        elif self.config['relative_to'].get() != 'playlist':
+            print(repr(self.config['relative_to'].get()))
```

Comment:
A `print` left over from debugging, perhaps?

Reply:
Oops, removed it. Sorry.